docs/platforms/javascript/common/tracing/configure-sampling/index.mdx

```diff
@@ -125,7 +127,7 @@ tracesSampler: (samplingContext) => {
 
 When multiple sampling mechanisms could apply, Sentry follows this order of precedence:
 
-- If `tracesSampler` is defined, its decision is used. Although the `tracesSampler` can override the parent sampling decision, most users will want to ensure their `tracesSampler` respects the parent sampling decision. 
+- If `tracesSampler` is defined, its decision is used. Although the `tracesSampler` can override the parent sampling decision, most users will want to ensure their `tracesSampler` respects the parent sampling decision.
```

Comment:
It might be hard to understand what to "parent sampling decision" is, if one is not super familiar with tracing.

Reply:
true, we can adjust this in a follow up maybe! Always hard to explain these things properly 😬